workers/unity/Assets/Tests/EditmodeTests/Performance/ThisShouldRun.cs

```diff
@@ -0,0 +1,73 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Subscriptions;
+using Improbable.Gdk.TestUtils;
+using NUnit.Framework;
+using Unity.PerformanceTesting;
+using UnityEngine;
+
+namespace Improbable.Gdk.EditmodeTests
+{
+    public class ThisShouldRun : MockBase
```

Comment:
What is the value of committing this to the codebase right now?

Reply:
This specific test - none

Even if we committed an actual test instead, there's no real value until we start pushing the results somewhere

I guess then do we even merge this PR (with a sensible test) or do we merge without running the perf-test CI step?

Reply:
I would prefer merging with a sensible (but simple) test if there is one readily apparent.